src/annotator/sidebar.coffee

```diff
@@ -71,7 +71,11 @@ module.exports = class Sidebar extends Host
       @onProfileRequest = serviceConfig.onProfileRequest
       @onHelpRequest = serviceConfig.onHelpRequest
 
-    @onLayoutChange = config.onLayoutChange
+    @onLayoutChangeHandlers = []
```

Comment:
In this approach, we're piggybacking on the existing config property `onLayoutChange`, assuming that PDF page-fitting would be _additive_ to any custom layout-change-handlers provided by config. It's also possible we'd want people to be able to disable PDF page-fitting via a config value. Anyway, there are about five thousand ways to slice this banana, so we can discuss...

Reply:
Another option here might be to leverage the event bus that the main annotator classes have (see `delegator.coffee` which is the EventEmitter-like class that has `publish` and `subscribe` methods). For example, the `sidebar` class could emit events when the sidebar layout changes and `PdfSidebar` or perhaps another observer could handle it and adjust the document as necessary. This might help with decoupling from the rest of the sidebar and to provide extensibility for other document viewers in future.